### PR TITLE
docs(865635): Added Ribbon Gallery Item UG with it's subtopics.

### PR DESCRIPTION
**Pull request by SenthilarasuBalu**
### Description
Need to add UG content for the Ribbon Gallery Item with it's subtopics.

**Gallery**

- Groups
  - Items
     - Content
     - Icon Css
     - HTML Attributes
     - Css class
     - Disabled
  - Header
  - Item Width
  - Item Height
  - Css Class
- Item Count
- Selected Item Index
- Popup Height
- Popup Width
- Template
- Popup Template
- Events
   -  Popup Open
   - Popup Close
   - Item Hover
   - Belore Item render
   - Before select
   - Select

### Screenshot
**Items**
image

**Content**
image

**Icon Css**
image

**HTML Attributes**
image

**Css class**
image
**Disabled**
image

**Template and Popup Template**
image

**Header**
image

**Item Width and Height**
image

**Css Class**
image

**Item Count**
image

**Selected Item Index**
image

**Popup Height and Width**
image

### CODE REVIEW

File: ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      {
+        header: "Gallery",
+        collections: [
+          {
+            items: [
+              {
+                type: "Gallery",
+                gallerySettings: {
+                  groups: [{
+                    header: 'Clipboard',
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js:14]: use Simple gallery examples, this is not real use-case @SenthilarasuBalu

File: ej2-javascript/code-snippet/ribbon/gallery/galleryIcon/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        collections: [
+          {
+            items: [
+              {
+                type: "Gallery",
+                gallerySettings: {
+                  groups: [{
+                    header: 'Clipboard',
+                    items: [{
+                            content: 'Cut',
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/code-snippet/ribbon/gallery/galleryIcon/index.js:16]: use any valid gallery icons case @SenthilarasuBalu .

File: ej2-javascript/code-snippet/ribbon/gallery/classCustomization/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        collections: [
+          {
+            items: [
+              {
+                type: "Gallery",
+                gallerySettings: {
+                  groups: [{
+                    header: 'Good, Bad and Neutral',
+                    items: [{
+                            content: 'Normal',
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/code-snippet/ribbon/gallery/classCustomization/index.js:16]: UI is not proper, apply some margins for the items like in SB's @SenthilarasuBalu

File: ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+var tabs = [
+  {
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js:2]: Keep any default simple items along with gallery @SenthilarasuBalu. To make that UI better.

File: ej2-javascript/code-snippet/ribbon/gallery/itemHeightAndWidth/index.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        collections: [
+          {
+            items: [
+              {
+                type: "Gallery",
+                gallerySettings: {
+                  groups: [{
+                    itemWidth: '100',
+                    itemHeight: '30',
+                    header: 'Title and Headings',
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/code-snippet/ribbon/gallery/itemHeightAndWidth/index.js:16]: Change this example to below @SenthilarasuBalu 

image

File: ej2-javascript/ribbon/ej2-javascript-ribbon-toc.html [hunk 6]
```diff
@@ -12,6 +12,7 @@
 <li><a href="/ej2-javascript/ribbon/resizing">Resizing</a></li>
 <li><a href="/ej2-javascript/ribbon/events">Events</a></li>
 <li><a href="/ej2-javascript/ribbon/accessibility">Accessibility</a></li>
+<li><a href="/ej2-javascript/ribbon/gallery-items">Gallery Items</a></li>
```
Comment (IndrajithSrinivasan) [inline ej2-javascript/ribbon/ej2-javascript-ribbon-toc.html:15]: Pull latest changes from development use below heading name and add next to Keytips sections @SenthilarasuBalu 

**Ribbon Gallery**

File: ej2-javascript/code-snippet/ribbon/gallery/classCustomization/index.js (hunk 3 above)
Comment (SenthilarasuBalu) [reply ej2-javascript/code-snippet/ribbon/gallery/classCustomization/index.js:50]: margin added.

File: ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js (hunk 4 above)
Comment (SenthilarasuBalu) [reply ej2-javascript/code-snippet/ribbon/gallery/beforeItemRender/index.js:2]: Clipboard items added.

File: ej2-javascript/code-snippet/ribbon/gallery/itemHeightAndWidth/index.js (hunk 5 above)
Comment (SenthilarasuBalu) [reply ej2-javascript/code-snippet/ribbon/gallery/itemHeightAndWidth/index.js]: sample updated.

File: ej2-javascript/ribbon/ej2-javascript-ribbon-toc.html (hunk 6 above)
Comment (SenthilarasuBalu) [reply ej2-javascript/ribbon/ej2-javascript-ribbon-toc.html]: Pulled the latest changes

**Review by IndrajithSrinivasan — APPROVED** [review]

**Pull request merged**